src/languages/es.js

```diff
@@ -775,19 +775,19 @@ export default {
     },
     additionalDetailsStep: {
         headerTitle: 'Detalles adicionales',
-        helpText: 'Necesitamos confirmar la siguiente información antes de que podamos procesar este pago.',
-        helpTextIdologyQuestions: 'Tenemos que preguntarte unas preguntas mas para terminar de verificar tu identidad',
+        helpText: 'Necesitamos confirmar la siguiente información antes de que podamos procesar el pago.',
```

Comment:
English says `this`, I think this is shown in the flow of attempting to pay a request?

Reply:
It's inferred that is referring to the payment that they are trying to perform but I can say `este` again. Up to you